python/cloud-run-django-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+steps:
+- id: docker build
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/${_PROJECT_ID}/${_SERVICE}-${COMMIT_SHA}', '.']
+
+- id: docker push
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['push', 'gcr.io/${_PROJECT_ID}/${_SERVICE}-${COMMIT_SHA}']
+
+- id: Deploy to Cloud Run
+  name: 'gcr.io/cloud-builders/gcloud'
+  args:
+  - run
+  - deploy
+  - ${_SERVICE}
+  - --image
+  - gcr.io/${_PROJECT_ID}/${_SERVICE}-${COMMIT_SHA}
+  - --region
+  - ${_REGION} # e.g. us-central1
+  - --platform
+  - managed
+  - --allow-unauthenticated
+
+images:
+- gcr.io/${_PROJECT_ID}/${_SERVICE}-${COMMIT_SHA}
+
+substitutions:
```

Comment:
Please add steps to test the app. Basic curl ping is good enough.
You can see my dotnet PR or Adam's Go PR and adapt yours accordingly.

Reply:
Will do.